exporter/elasticsearchexporter/bulkindexer.go

```diff
@@ -383,6 +484,81 @@ func flushBulkIndexer(
 		}
 		failedDocsInputLogger.Debug("failed to index document; input may contain sensitive data", fields...)
 	}
+	if stat.Indexed > 0 {
+		tb.ElasticsearchDocsIndexed.Add(
+			ctx,
+			stat.Indexed,
+			metric.WithAttributeSet(attribute.NewSet(
+				attribute.String("outcome", "success"),
+			)),
+		)
+	}
+	if tooManyReqs > 0 {
+		tb.ElasticsearchDocsIndexed.Add(
+			ctx,
+			tooManyReqs,
+			metric.WithAttributeSet(attribute.NewSet(
+				attribute.String("outcome", "too_many"),
+			)),
+		)
+	}
+	if clientFailed > 0 {
+		tb.ElasticsearchDocsIndexed.Add(
+			ctx,
+			clientFailed,
+			metric.WithAttributeSet(attribute.NewSet(
+				attribute.String("outcome", "failed_client"),
+			)),
+		)
+	}
+	if serverFailed > 0 {
+		tb.ElasticsearchDocsIndexed.Add(
+			ctx,
+			clientFailed,
```

Comment:
```suggestion
			serverFailed,
```
typo here. any test coverage?